src/app/plugins/platform/JavaScript/templates/tasks/webpack.js

```diff
@@ -0,0 +1,16 @@
+const webpack = require('webpack');
+
+module.exports = function webpackBuild() {
```

Comment:
I'd like to see this as an `async` function. You can use [`util.promisify()`](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_original) to convert the `webpack` function into a function that returns a promise for the Webpack stats object. (Don't forget to `await` the result.)